### ISSUE TITLE
Make "-dont_restart_as_admin" the default behaviour

### ISSUE THREAD

**Issue by Kein** [GameTechDev/PresentMon#96]
I have a process `MyProcess.exe` started and running under user.

```
PresentMon64-1.5.2.exe -dont_restart_as_admin -terminate_after_timed -delay 20 timed 20 -process_name MyProcess.exe
warning: PresentMon requires elevated privilege in order to query processes
    started on another account, so these processes won't be targetted by name
    and will be listed as '<error>'.  Further, there may be tracking errors
    near process termination, and if they are targetted
    -terminate_on_proc_exit won't work.
error: failed to start session (error=5).
```

There is no reason why you can't target userprocess without elevating to admin right? It should only fail if PM cant find specific process instead of failing statically and pre-emptively. If you are afraid user wont understand the reason then just make error description comprehensive, instead of just "No specific process found", add an extra info pointing out to the fact that PresentMon needs to be run under Admin or specific User if you want to capture all processes or specific, belonging to aforementioned user.

**Comment by JeffersonMontgomery-Intel** [comment]
Yes, you shouldn't need admin to track your own processes.  PresentMon is printing the warning and then continuing, but it ends up failing to create a ETW trace session later on.  I'm not sure why that's happening, and it's not reproducing on my system:

```
PresentMon64-1.5.2.exe -no_csv -dont_restart_as_admin -terminate_after_timed -delay 20 -timed 20 -process_name d3d11_triangle.exe
warning: PresentMon requires elevated privilege in order to query processes
    started on another account, so these processes won't be targetted by name
    and will be listed as '<error>'.  Further, there may be tracking errors
    near process termination, and if they are targetted
    -terminate_on_proc_exit won't work.
d3d11_triangle.exe[27968]: 000002806848F1C0 (DXGI): SyncInterval=1 Flags=0 16.67 ms/frame (60.0 fps, 60.0 fps displayed, 49.01 ms latency) Composed: Copy with GPU GDI
```

Could you try with the latest version 1.6.0 and let me know if the problem is still there?

**Comment by Kein** [comment]
Oh, didnt check releases, my bad. The issue is not present on 1.6.0, yeah

**Comment by JeffersonMontgomery-Intel** [comment]
Cool, yeah I cleaned some of that up recently so glad it's better.  Thanks for reporting.

**Comment by Kein** [comment]
Why is elevation enabled by default? How often do you need to capture 3D-performing process running under system or different user? Imho, it should run normally by default and user has to have a choice to elevate it directly via system means rather than being forced to avoid elevating it **by default**.

**Comment by jenatali** [comment]
Some system processes, notably `dwm.exe`, actually run in a separate user account. That's the primary reason for doing it by default if I remember correctly.

**Comment by Kein** [comment]
Sure, it runs under `Window Manager\DWM-1` but that's why you elevate for that manually, instead it being forced. Unless you want to imply massive majority of PM users use it to capture DWM and other system processes. I could see this being true earlier back then but as of now I haven't seen it being used anywhere outside of 3D processes running under user (games, duh).

**Comment by jenatali** [comment]
Sure, if you specify a process name I could see skipping default elevation, but if you don't, then it should capture the whole system, including processes from other users. For the record I don't feel strongly here, just commenting on why it was done.

**Comment by Kein** [comment]
Yeah I'm just not particularly of a fan it is being forced to elevate by default. I think it just not very reasonable since it can naturally work with default limited approach just fine and if user needs it he can elevate it. It would be nice if this was revised in future version.

**Comment by Kein** [comment]
```
g:\Programs\CapFrameX\PresentMon>PresentMon-1.6.0-x64.exe -dont_restart_as_admin -terminate_after_timed -delay 10 -process_name CyberShadow.exe -timed 10
warning: PresentMon requires elevated privilege in order to query processes started
    on another account.  Without it, those processes will be listed as '<error>'
    and they can't be targeted by -process_name nor trigger -terminate_on_proc_exit.
error: failed to start trace session (access denied).
```

It is back.

**Comment by jdm3** [comment]
OK, I think I know what the issue is.

Even for tracing your own processes, PresentMon requires either administrative privileges or to be run by a user in the "Performance Log Users" user group.   I'm investigating how to detect this to improve the error, and will document it in the readme, but for now to add your user account to the group, follow these steps:

1. Run compmgmt.msc as administrator.
3. In the "Computer Management" window, expand "System Tools", expand "Local Users and Groups", and then click "Groups".
4. Double-click "Performance Log Users", and then click "Add".
5. In the "Enter the object names to select" text box, type the name of the user account or group account that you want to add, and then click "OK".
6.  Sign out and log back in for the changes to take effect.

**Comment by Kein** [comment]
What the reason for 
>"Performance Log Users" user group. 
requirement?

**Comment by jenatali** [comment]
PresentMon uses ETW (Event Tracing for Windows), which requires either administrator permissions or for the user to be in that group. It's a requirement imposed by Windows.

**Comment by Kein** [comment]
> PresentMon uses ETW (Event Tracing for Windows),

I mean, what for this specific requirement?

**Comment by jenatali** [comment]
From the readme:

> PresentMon is a tool to capture and analyze ETW events related to swap chain presentation on Windows.

That's just the definition of what this tool is...

**Comment by JeffersonMontgomery-Intel** [comment]
Yeah, it's totally fundamental: presentmon get's all of its metrics data through ETW, which gives it insight into Windows graphics systems as well as portability across GPU vendors.  perflog users provides much less access than administrators, but it is the minimum requirement.
